packages/react-native-reanimated/src/createAnimatedComponent/PropsFilter.tsx

```diff
@@ -67,16 +67,32 @@ export class PropsFilter implements IPropsFilter {
         has('workletEventHandler', value) &&
         value.workletEventHandler instanceof WorkletEventHandler
```

Comment:
You can use your new `isWorkletEventHandler` method here

Reply:
It's not exported anywhere but I'll think of it!